tests/integration/conftest.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import configparser
+import re
+
+import pytest
+import pytest_asyncio
+import pytest_operator.plugin
+
+
+@pytest.fixture
+def openstack_environment(request):
+    """Parse the openstack rc style configuration file from the --openstack-rc argument
+
+    Return a dictionary of environment variables and values.
+    """
+    rc_file = request.config.getoption("--openstack-rc")
+    with open(rc_file) as f:
+        rc_file = f.read()
+    rc_file = re.sub("^export ", "", rc_file, flags=re.MULTILINE)
+    openstack_conf = configparser.ConfigParser()
+    openstack_conf.read_string("[DEFAULT]\n" + rc_file)
+    return {k.upper(): v for k, v in openstack_conf["DEFAULT"].items()}
+
+
+@pytest.fixture(scope="module")
+def content_cache_image(pytestconfig: pytest.Config):
+    """Get the content-cache image."""
+    value: None | str = pytestconfig.getoption("--content-cache-image")
+    assert (
+        value is not None
+    ), "please specify the --content-cache-image command line option"
```

Comment:
Would it be worth giving more information on what that image should be?

Reply:
Since this is the docker image the charm uses as a resource, is it really necessary to specify?